kbs/src/plugins/implementations/splitapi/manager.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2024 by IBM Corporation
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{anyhow, Context, Result};
+use std::{sync::Arc};
+use serde::Deserialize;
+use std::{
+    fs,
+    path::{Path, PathBuf},
+};
+use std::sync::Mutex;
+use lazy_static::lazy_static;
+
+use super::backend::{SplitAPIBackend, SandboxParams};
+use super::mapper::{SandboxDirectoryMapper, SandboxDirectoryInfo};
+use super::generator::{CredentialBundle, ServerCredential};
+
+
+pub const DEFAULT_PLUGIN_DIR: &str = "/opt/confidential-containers/kbs/plugin/splitapi";
+pub const SANDBOX_DIRECTORY_MAPPING_FILENAME: &str = "sandbox-credential-mapping.json";
+
+
+// Use lazy_static to initialize the SANDBOX_DIRECTORY_MANAGER only once
+lazy_static! {
+    static ref SANDBOX_DIRECTORY_MAPPER: Arc<Mutex<Option<SandboxDirectoryMapper>>> = Arc::new(Mutex::new(None));
+}
```

Comment:
Could we move this static variable directly into `CertManager` as a member? s.t.
```rust
pub struct CertManager {
    pub plugin_dir: String,
    pub mapping_filename: String,
    mapper: Arc<Mutex<SandboxDirectoryMapper>>,
}
```

Also, could we use [tokio::sync::Mutex](https://docs.rs/tokio/1.41.1/tokio/sync/struct.Mutex.html#examples) rather than `std::sync::Mutex`? The async version would promote CPU efficiency.